exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/profile/ProfileResources.java

```diff
@@ -77,12 +77,14 @@
    */
   public static String getPrettyDuration(long startTimeMillis, long endTimeMillis) {
     long durationInMillis = (startTimeMillis > endTimeMillis ? System.currentTimeMillis() : endTimeMillis) - startTimeMillis;
-    long hours = TimeUnit.MILLISECONDS.toHours(durationInMillis);
+    long days = TimeUnit.MILLISECONDS.toDays(durationInMillis);
```

Comment:
@paul-rogers , The format in DRILL-5230 is compacted to an approximation (i.e. we retain the 2 largest units of time) so as to fit within the cells of a table. 
This format is more verbose as it gives the complete picture, down to the smallest unit of time. Do we want to still merge the two into a single utility method?